src/main/java/bfk/brickbreaker/Direction.java

```diff
@@ -0,0 +1,9 @@
+package bfk.brickbreaker;
+
+public enum Direction {
```

Comment:
These are collision directions?

Not sure what `BOTTOMPADDLE` means

Reply:
It hits the paddle on the bottom. Trying to differentiate between that and when it hits the actual bottom (falls off the screen & the user 'loses' or moves on to the next round)